cli/src/auth.ts

```diff
@@ -72,16 +74,40 @@ program
         console.log("Tree Leaf: " + tree_leaf);
         console.log("Tree Proof: " + tree_proof);
 
-        let payload: Payload = {
-            amount,
-            destinationKey: new PublicKey(destination_address),
-            derivedKeySeeds: null,
-            treeMatchLeaf: null,
-        };
-        payload.amount = amount;
-        let result = await validateOperation(connection, payer, name, operation, payload);
-        console.log("Result: " + result);
+        // let payload: Payload = {
+        //     amount,
+        //     destinationKey: new PublicKey(destination_address),
+        //     derivedKeySeeds: null,
+        //     treeMatchLeaf: null,
+        // };
+        // payload.amount = amount;
+        // let result = await validateOperation(connection, payer, name, operation, payload);
+        // console.log("Result: " + result);
```

Comment:
This should be uncommented out and fixed, right?

Reply:
I was going to do this in a separate PR.

Reply:
ok sounds good